### ISSUE TITLE
The difference between the paper and the code

### ISSUE THREAD

**Issue by Tramac** [junfu1115/DANet#45]
Given a local feature A, first feed it into a convolution layers with batch normalization and ReLU layers to generate two new feature maps B and C.(in paper)
But, there is no BN and ReLU in code ```self.query_conv = Conv2d(in_channels=in_dim, out_channels=in_dim//8, kernel_size=1)``` and ```self.key_conv = Conv2d(in_channels=in_dim, out_channels=in_dim//8, kernel_size=1)```, in  ```PAM_Module```.
Which is right?

**Comment by junfu1115** [comment]
@Tramac, Thank you very much. We will correct the statement in the paper.

**Comment by Tramac** [comment]
Thank you for your reply！Another difference: operation ```energy_new = torch.max(energy, -1, keepdim=True)[0].expand_as(energy)-energy``` in ```CAM_Module```  is not indicated in paper.

**Comment by junfu1115** [comment]
@Tramac thanks, the operation is adopted for improving convergence during training, which is not mentioned in the paper

**Comment by Tramac** [comment]
Understood, thank you!
